robustbench/model_zoo/architectures/sodef_layers.py

```diff
@@ -0,0 +1,143 @@
+import math
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import geotorch
+from torch.nn.parameter import Parameter
+
+from robustbench.model_zoo.architectures.dm_wide_resnet import CIFAR10_MEAN, CIFAR10_STD, \
+    DMWideResNet, Swish
+
+
+class Identity(nn.Module):
+    def __init__(self):
+        super(Identity, self).__init__()
```

Comment:
Python's 3.x syntax can be more concise with `super().__init__()`

Reply:
I'd leave those as they are for now